elasticdl/python/master/pod_manager.py

```diff
@@ -145,6 +149,78 @@ def create_pod_manager(args):
     return pod_manager
 
 
+PodStateFlow = namedtuple(
+    "PodStateFlow",
+    ("from_status", "to_status", "event_type", "phase", "should_relaunch"),
+)
+
+POD_STATE_FLOWS = [
```

Comment:
Can you add a docstring with DAG to describe the relationship of different `PodStateFlow`?

Reply:
Good idea. Add the link to the DAG in the github issue. Drawing the graph using docstring is not easy.